metadata/src/test/java/org/apache/kafka/image/FeaturesImageTest.java

```diff
@@ -96,6 +121,20 @@ public void testImage2RoundTrip() {
         testToImage(IMAGE2);
     }
 
+    @Test
+    public void testImage3RoundTrip() {
+        testToImage(IMAGE3);
+    }
+
+    @Test
+    public void testApplyDelta2() {
+        assertEquals(IMAGE3, DELTA2.apply());
+        // check image2 + delta2 = image3, since records for image2 + delta2 might differ from records from image3
+        List<ApiMessageAndVersion> records = getImageRecords(IMAGE3);
```

Comment:
`s/IMAGE3/IMAGE2`

I wonder why this test is still passing with this mistake?  Probably not a big deal, but still might be good to satisfy yourself that it makes sense that it would pass like this.

Reply:
thanks for catching this, think it makes sense to me that this would pass - the records from delta2 applied ontop of image3 still come out to image3